setlexsem/generate/generate_prompts.py

```diff
@@ -67,23 +122,136 @@ def create_prompt(
         prompt = get_prompt(
             A,
             B,
-            prompt_config,
+            prompt_config_ready,
             add_roles=add_roles,
         )
 
-        ground_truth = get_ground_truth(prompt_config.operation, A, B)
+        ground_truth = get_ground_truth(prompt_config_ready.operation, A, B)
 
         prompt_and_ground_truth.append(
             {
                 "prompt": prompt,
                 "ground_truth": ground_truth,
-                **prompt_config.to_dict(),
+                **prompt_config_ready.to_dict(),
             }
         )
 
     return prompt_and_ground_truth
 
 
+def generate_prompts(
```

Comment:
We should consider favoring `make` or `create` over `generate`. It is a generated, synthetic dataset, but "generative" is super overloaded now.